resources/translations/en.edn

```diff
@@ -300,6 +300,7 @@
   :link {:download-file "Download file"
          :opens-in-new-window "Opens in a new window"}
   :login {:alternative "Other login options"
+          :auth0-text "Login with Auth0"
```

Comment:
I don't think the user should know anything about Auth0. From the user's point of view he logs in using his Google/Facebook/etc. account.

Reply:
Auth0 does provide direct credentials too, social media accounts are not mandatory. Also this UI is really meant for development and not end users, similarly how fake-shibboleth has a Haka login logo even though Haka is not used. Actual user would use the Elixir AAI once we have the necessary information.

If we do want to have a more polished Auth0 support for end users, what would be a sensible text for that? "Login with Google/Facebook/Github/Auth0"? Or we could use the Auth0 Universal Login widget (https://auth0.com/docs/libraries/when-to-use-lock for looks) or directly have buttons for Google, Facebook etc. (we would still always use Auth0, but each link would specify the actual auth provider).

Reply:
"Login with OpenID" could be better, since it could be some other OpenID provider as well.

Reply:
Or just one big Login button.